Kickstarter-iOS/Views/Controllers/CancelPledgeViewController.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class CancelPledgeViewController: UIViewController {
+  private let viewModel: CancelPledgeViewModelType = CancelPledgeViewModel()
+
+  // MARK: - Lifecycle
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    _ = self
+      |> \.title %~ { _ in Strings.Cancel_pledge()
+      }
+  }
+
+  // MARK: - Configuration
+
+  internal func configure(with _: Project, backing _: Backing) {}
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.view
+      |> \.backgroundColor .~ UIColor.ksr_grey_400
```

Comment:
Should we use `checkoutBackgroundStyle` to stay consistent?

Reply:
It's not the right color 😭 we really need to do a sweep of all our background colors and align on just one if possible.